File: Python/8py 1/format_money.py
"""
|--------------------------------------------------------------------------
| Dollars and Cents
|--------------------------------------------------------------------------
|
"""

def format_money(amount):
   x = '{:.2f}'.format(amount)
   return f'${x}'


def format_money(amount):
   return '${:.2f}'.format(amount)


def format_money(amount):
   return '$%0.2f' % amount


def format_money(amount):
   return f'${amount:.2f}'


print(format_money(39.99), '$39.99')

File: JavaScript/8js 1/formatMoney.js
/*
|--------------------------------------------------------------------------
| Dollars and Cents
|--------------------------------------------------------------------------
|
*/

function formatMoney(amount) {
   return `$${String(amount.toFixed(2))}`
}


function formatMoney(amount) {
   return '$' + amount.toFixed(2);
}


function formatMoney(amount) {
   const newAmount = (amount).toLocaleString('en-US', {
      style: 'currency',
      currency: 'USD'
   }).replace(/,/g, "")
   return newAmount
}


console.log(formatMoney(39.99), '$39.99');